storybook/index.css

```diff
@@ -68,14 +24,6 @@
   -moz-osx-font-smoothing: grayscale;
 }
 
-body {
-  margin: 0;
-  display: flex;
-  place-items: center;
-  min-width: 320px;
-  min-height: 100vh;
-}
-
```

Comment:
> [!NOTE]
> Removed these styles because it was affecting our storybook previews. Screenshot below.

![capture -Arc-2025-01-08@2x](https://github.com/user-attachments/assets/31e8c516-0ce4-4acf-9174-fadd9cab4662)